app/src/main/res/values/strings.xml

```diff
@@ -55,6 +56,9 @@
     <string name="reset_name">Reset history</string>
     <string name="keyboard_desc">Always display keyboard at startup</string>
     <string name="keyboard_name">Display keyboard</string>
+    <string name="root_mode_desc">Enable root features if available</string>
```

Comment:
maybe explain why someone would want this? E.g. "Enable root features (for hibernating apps)"